content/04.study.md

```diff
@@ -424,6 +424,92 @@ summarized above also apply to interfacial contact prediction for protein
+They found that MHCnuggets — the recurrent neural network — was by far the fastest training among the top performing methods.
+In predicting interactions between proteins, deep learning has achieved state-of-the-art results and shows promise to overcome previous challenges in the field.
+
+### PPI networks and graph analysis
+
+The abundance of large networks of biological data presents a different approach to computational prediction.
+Because interacting proteins are more likely to share a similar function, the connectivity of a PPI network itself can be a valuable information source for the prediction of function [@doi:10.1038/msb4100129].
+To incorporate higher-order network information, it is necessary to find a lower-level embedding of network structure that preserves this higher-order structure.
+Rather than use hand-crafted network features, deep learning shows promise for the automatic discovery of predictive features within networks.
+For example, Navlakha [@doi:10.1162/NECO_a_00294] showed that a deep autoencoder was able to compress a graph to 40% of its original size, while being able to reconstruct 93% of the original graph's edges.
+The deep learning method had greater reconstruction accuracy than comparable dimensionality-reduction methods such as PCA with SVD.
+While the activity of some hidden layers correlated with several popular hand-crafted network features, this work showed that deep learning can effectively reduce graph dimensionality while retaining much of its structural information.
+
+An important challenge in PPI network prediction is the task of combining different networks and types of networks.
+A way of working with different network types was shown by Gligorijevic et al., [@doi:10.1101/223339] who developed a multimodal deep autoencoder, deepNF, to find a feature representation common among several different PPI networks.
+This common lower-level representation allows for the combination of various PPI data sources towards a single predictive task.
+An SVM classifier trained on the compressed features from the middle layer of the autoencoder outperformed previous methods in predicting protein function.
+The key advancement of this method is the use of deep learning to incorporate higher-order network information for protein function prediction.
+
+Hamilton et al. addressed the issue of large, heterogeneous, and changing networks with an inductive approach called GraphSAGE [@arxiv:1706.02216v2].
+By finding node embeddings through learned aggregator functions which describe the node and its neighbors in the network, the GraphSAGE approach allows for the generalization of the model to unknown nodes.
+Generalization to unseen nodes is especially useful for PPI networks, as these networks represent various types of interactions between proteins in a variety of species, and they can be updated frequently.
```

Comment:
I don't think I'm following this.  Isn't an unseen node a new protein in a PPI network?  Do we encounter new proteins?  Or is the idea that a trained model generalizes to new graphs?